eni.tf

```diff
@@ -14,14 +14,14 @@ resource "aws_network_interface" "additional" {
 }
 
 resource "aws_network_interface_attachment" "additional" {
-  count                = "${local.additional_ips_count}"
+  count                = "${local.additional_ips_count * var.instance_count}"
   instance_id          = "${aws_instance.default.id}"
-  network_interface_id = "${aws_network_interface.additional.*.id[count.index]}"
+  network_interface_id = "${element(aws_network_interface.additional.*.id, count.index / var.instance_count)}"
```

Comment:
We should probably call `floor(count.index / var.instance_count)` to be explicit

Reply:
okay sweet